ignite/pkg/cosmosmetric/adapter/postgres/postgres.go

```diff
@@ -0,0 +1,347 @@
+package postgres
+
+import (
+	"context"
+	"database/sql"
+	"embed"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosclient"
+	"github.com/ignite-hq/cli/ignite/pkg/cosmosmetric/query"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+
+	_ "github.com/lib/pq" // required to register postgres sql driver
+)
+
+const (
```

Comment:
Instead of a single const block, can we group the related ones?

One group for DefaultX,

A single const for adapterType,

One group for sqlX.